scripts/operations/configuration/refresh_master_storage_rack_resiliency_config.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+Usage: refresh_master_storage_rack_resiliency_config.py
+
+This script modifies the CFS component status of both Master and Storage NCNs
+to ensure the Rack Resiliency playbook layer is properly refreshed.
+
+Behavior:
+  - If NONE of the Master or Storage NCNs have the Rack Resiliency layer → abort.
```

Comment:
This script should have the same logic as the iscsi script to abort if any of the components is currently being configured.

UPDATE: I see that you actually do, but it isn't mentioned here. This goes along with my previous comment about the docstring.